.github/workflows/Bundle.yml

```
@@ -0,0 +1,66 @@
+        run: |
+          cd ${{ github.workspace }}/build
+          rm -fr ./*
+          cmake -DPICO_SDK_PATH=${{ github.workspace }}/pico-sdk -DPICOSTATION_VARIANT=picostation_pico2 .. 
+          cmake --build .
+          cp ${{ github.workspace }}/build/picostation_pico2.uf2 ${{ github.workspace }}/output/picostation_pico2.uf2
+
+      - name: List Files
+        run: ls ${{ github.workspace }}/build
+
+      - name: Create Draft Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)